packages/components/note/src/Note.tsx

```diff
@@ -36,7 +36,7 @@ export type NoteInternalProps = CommonProps & {
   /**
    * Sets title in the Note
    */
-  title?: string;
+  title?: string | React.ReactNode;
```

Comment:
```suggestion
  title?: React.ReactNode;
```

strings are also ReacNode so no need to do the `|` thing